.github/ISSUE_TEMPLATE/maintainerRequest.yml

```diff
@@ -0,0 +1,83 @@
+name: Maintainer Request
+description: Request membership to help maintain this repository.
+title: Maintainer Request
+labels: ['maintainer-request']
+projects: ['repo-management']
+assignees:
+  - kilasuit
```

Comment:
This should be for the core maintainer/maintainers and coudl perhaps be removed due to the codeowners file